packages/pkgs/components/Modal/index.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+
+import Box from '../Box';
+
+export type Props = {
+    children?: React.Node,
+    open?: boolean,
+};
+
+const Modal: React$AbstractComponent<Props, HTMLElement> = React.forwardRef<Props, HTMLElement>(({
+  children = null,
+  open = true,
+}: Props, ref) => {
+  const containerRef = React.useRef();
+  const activeRef = ref || containerRef;
+
+  const styles = {
+    container: {
+      width: '50%',
```

Comment:
Does this modal actually pop out? Try something like position fixed, and then give it a white box